### PR TITLE
fix url of debug markers sample

### PR DESCRIPTION
**Pull request by haxpor**

### CODE REVIEW

**Comment by baldurk** [comment]
Thanks for catching this broken URL. I think it's probably a better idea to drop the direct link to the debug markers sample since it might break again in future, and just link to the github repo (which is much less likely to change) and say that it's the `debugmarker` sample.

If you'd like to update this PR that would be welcome, otherwise I can make the fix myself.

**Comment by haxpor** [comment]
I can do it. Do you mean it's better to link directly to debugmarker sample i.e. https://github.com/SaschaWillems/Vulkan/tree/master/examples/debugmarker ?

**Comment by baldurk** [comment]
The branch or path might still change, and there's not too much value realy in linking directly to the example vs. just naming it and letting someone following the repository link find it. Something like this:

```rst
 For this tutorial we will be using the ``debugmarker`` sample from `Sascha Willems' Vulkan samples repository <https://github.com/SaschaWillems/Vulkan>`_.. 
```

**Comment by haxpor** [comment]
Ok I get it now. Updated.

File: docs/getting_started/quick_start.rst [hunk 1]
```diff
@@ -3,7 +3,7 @@ Quick Start
 
 This document serves as a brief introduction to how to use RenderDoc to capture and analyse your application. It doesn't go into much detail about exactly what functionality is available for digging into things, as this can be found elsewhere. Instead it focuses on just the workflow from start to finish and roughly where to look for different things.
 
-For this tutorial we will be using the `debugmarker <https://github.com/SaschaWillems/Vulkan#vk_ext_debug_marker>`_. sample from `Sascha Willems' Vulkan samples repository <https://github.com/SaschaWillems/Vulkan>`_..
+For this tutorial we will be using the `debugmarker` sample from `Sascha Willems' Vulkan samples repository <https://github.com/SaschaWillems/Vulkan>`_..
```
Comment (baldurk) [inline docs/getting_started/quick_start.rst]: The restructuredtext syntax for monospace is ` `` ` double backticks rather than single backticks.

Comment (haxpor) [reply docs/getting_started/quick_start.rst]: Sorry, I'm carelessly looking pass your example. It should be fine now I think.

**Pull request merged**